hxd/Window.hl.hx

```diff
@@ -5,6 +5,12 @@ import hxd.Key in K;
 #error "You shouldn't use both -lib hlsdl and -lib hldx"
 #end
 
+#if hlsdl
+typedef DisplayMode = sdl.Window.DisplayMode;
+#elseif hldx
+typedef DisplayMode = dx.Window.DisplayMode;
+#end

```

Comment:
Oh yeah, this will fail on usesys (`DisplayMode` undefined). Should it just be an enum on that target?